src/components/snackbar/examples/snackbar-persistent-non-dismissible.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, Element, State, h } from '@stencil/core';
+
+/**
+ * Persistent and non-dismissible
+ * In some scenarios, a Snackbar that is persistent
+ * can simultaneously be non-dismissible.
+ *
+ * This is good for handling cases in which the system tries to
+ * communicate an ongoing background process or a status that the user
+ * cannot take an action on, but should be kept informed about.
+ *
+ * In such cases, you should programmatically close the Snackbar
+ * when the process is completed or the status changes!
```

Comment:
@jgroth I think this is were we got stuck last time if I remember correctly. How to close/dismiss a snackbar programmatically?

Reply:
It's not possible with how the component API is right now, we would have to add either an `open` property to replace the `show` method, or add a `hide` method in addition to the `show` method we already have.

Reply:
I can add. But I guess you prefer adding a `hide` method avoid breaking change, right?

Reply:
I would prefer the `open` property approach :smile: We can deprecate the `show` method to not make it breaking